config/default/kustomization.yaml

```diff
@@ -32,7 +32,7 @@ patchesStrategicMerge:
 
 # Mount the controller config file for loading manager configurations
 # through a ComponentConfig type
-#- manager_config_patch.yaml
+- manager_config_patch.yaml
```

Comment:
This loses the args set through `manager_auth_proxy_patch.yaml`.

Can you add `--zap-log-level=2` to `manager_config_patch.yaml`?

Reply:
done.